docs/manual.rst

```diff
@@ -2007,30 +2007,66 @@ For example, here is an identity function applicable to both types of input
 
     g2 = transform(id_func, g1)
 
-A partially applied transform function from pyproj satisfies the requirements
-for `func`.
+
+If using `pyproj>=2.1.0`, the preferred method to project geometries is:
 
 .. code-block:: python
 
+    import pyproj
+    import pyproj.crs
+
+    from shapely.geometry import Point
     from shapely.ops import transform
+
+    wgs84_pt = Point(-72.2495, 43.886)
+
+    wgs84 = pyproj.crs.CRS('epsg:4326')
+    utm = pyproj.crs.CRS('epsg:32618')
+
+    project = pyproj.Transformer.from_crs(wgs84, utm, always_xy=True).transform
+    utm_point = transform(project, wgs84_pt)
+
+Otherwise, a partially applied transform function from pyproj also satisfies the requirements
+for `func`.
```

Comment:
I am not fully sure that it is best to show both. As I think most people should use the above approach, also showing the below might only be confusing. Or maybe at least give a stronger indication the first is preferred?

Reply:
Yeah I think you're right, I'm going to remove the second example and not even bother showing a "not preferred" way to do it.